docker/docker_image_src.go

```diff
@@ -67,6 +67,7 @@ func newImageSource(ctx context.Context, sys *types.SystemContext, ref dockerRef
 		logrus.Debugf("Trying to access %q", pullSource.Reference)
 		s, err := newImageSourceAttempt(ctx, sys, ref, pullSource)
 		if err == nil {
+			sys.DockerLogPullSource = s.physicalRef.ref.Name()
```

Comment:
Hum. That’s certainly an inventive way to not break the API, but it’s too surprising a way to be maintainable. Generally code assumes that things like https://github.com/containers/image/blob/122b16af75d13b9dacda1b04111fa7fdffde30c5/docker/docker_image_src.go#L108-L111 are fine to do.